multiapps-controller-process/src/main/java/module-info.java

```diff
@@ -50,6 +50,8 @@
     requires spring.core;
     requires spring.security.oauth2;
     requires spring.web;
+    requires reactor.netty;
+    requires io.netty.handler;
```

Comment:
Why are these needed?

Reply:
Error occurred during initialization of boot layer
java.lang.module.FindException: Module reactor.netty not found, required by com.sap.cloudfoundry.client.facade
They are required as far as I understood